nemo/collections/multimodal/models/multimodal_llm/neva/neva_model.py

```diff
@@ -1199,6 +1218,8 @@ def sharded_state_dict(self, prefix: str = ''):
 
         original_sharded_state_dict = super().sharded_state_dict()
         keys_to_keep = self.get_keys_to_keep()
+        #TODO: this breaks. It can't find this tensor in the sharded dict, probably the reason why its not loaded.
+        # keys_to_keep.append("model.lisa_sam.sam.prompt_encoder.pe_layer.positional_encoding_gaussian_matrix")
```

Comment:
@anmolgupt to fix prompt encoder get dense PE, I believe we have to make it sharded so that it can read the tensor from the checkpoint. It's in the checkpoint, but it's not loaded due to this.